src/main/drivers/light_led.h

```diff
@@ -62,6 +40,22 @@ PG_DECLARE(statusLedConfig_t, statusLedConfig);
 #define LED2_OFF                 ledSet(2, false)
 #define LED2_ON                  ledSet(2, true)
 
+// use dummy functions for unittest
+#if defined(USE_VIRTUAL_LED)
+
+// ledInit is missing intentionally
+static inline void ledToggle(int led) { UNUSED(led); }
+static inline void ledSet(int led, bool state) { UNUSED(led); UNUSED(state); }
+
+#else
```

Comment:
Can't the dummy functions be defined in the unit test?

Reply:
This is also used for SITL (and technically any target can disable HW leds).
unittest stubs are ugly, IMO using USE_VIRTUAL_LED is slightly better in current state. It would change if isolated, reusable stubs are implemented